enclave-runner/src/usercalls/mod.rs

```diff
@@ -1058,7 +1058,11 @@ impl EnclaveState {
         p4: u64,
         p5: u64,
     ) -> StdResult<(u64, u64), failure::Error> {
-        let thread = enclave.threads_queue.pop().expect("threads queue empty");
+        let thread = executor::block_on(async {
```

Comment:
We should not be mixing executors (tokio and futures), use a tokio runtime here similar to `fn main_entry`